### PR TITLE
Profile pop up

### LINKED ISSUE #254: See user profile when we click on a post
Acceptance criteria:
- See a pop-up showing the user profile when we click on their avatar or display name
- This pop-up should contain both the Centauri score of the user, and their username

### PR DESCRIPTION
**Pull request by JoachimFavre**
Fixes #254. 

This PR adds a pop-up that shows the username and the Centauri score of a user when we click on their name or their avatar. This is done in the class `PublicationHeader`, which is used in the feed overview, in the post page and in the comment headers. Therefore, this works on all of those cases for free (encapsulation is so great).
<p align="center">
  
  
</p>

Acceptance criteria:
- [x] See a pop-up showing the user profile when we click on their avatar or display name
- [x] This pop-up should contain both the Centauri score of the user, and their username
- [x] Update the penpot accordingly

### CODE REVIEW

File: lib/views/components/content/publication_header/user_profile_pop_up.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/user_avatar_details.dart";
+import "package:proxima/views/components/content/user_avatar/user_avatar.dart";
+import "package:proxima/views/helpers/key_value_list_builder.dart";
+
+class UserProfilePopUp extends StatelessWidget {
```
Comment (camillelnne) [inline lib/views/components/content/publication_header/user_profile_pop_up.dart:6]: Could you add some documentation for this class please?

**Review by camillelnne — COMMENTED** [review]
Love the usernames in the mock data :)
The testing is very thorough.

Just missing some documentation in your new class.

File: lib/views/components/content/publication_header/user_profile_pop_up.dart (hunk 1 above)
Comment (JoachimFavre) [reply lib/views/components/content/publication_header/user_profile_pop_up.dart:10]: Ah yes indeed, this is a good idea. Fixed in 71237ce24f39bbebef736f75831ca6ac92c11db7.

**Review by camillelnne — APPROVED** [review]
Thanks for the change!
LGTM :)

Penpot approved too, good job!

**Pull request merged**